src/utils.py

```diff
@@ -21,3 +22,15 @@ def get_image(image_dir_path: pathlib.Path) -> np.array:
 def remove_output_dir(output_dir: pathlib.Path) -> None:
     if output_dir.exists():
         shutil.rmtree(output_dir)
+
+
+def get_numcodec_shuffle(shuffle: str) -> int:
```

Comment:
```suggestion
def get_numcodec_shuffle(shuffle: Literal['shuffle', 'noshuffle', 'bitshuffle']) -> int:
```

Reply:
Thanks @dstansby ! I've added `Literal` typing for `shuffle` and `zarr_spec` now :smile: